### PR TITLE
Update datamodel-properties.asciidoc

### PR DESCRIPTION
**Pull request by anasmi**
Fixes : https://github.com/vaadin/framework/issues/11095

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/framework/11223)

### CODE REVIEW

File: documentation/datamodel/datamodel-properties.asciidoc [hunk 1]
```diff
@@ -299,7 +299,7 @@ class MyConverterFactory extends DefaultConverterFactory {
 }
 
 // Use the factory globally in the application
-Application.getCurrentApplication().setConverterFactory(
+UI.getCurrent().getSession().setConverterFactory(.setConverterFactory(
```
Comment (knoobie) [inline documentation/datamodel/datamodel-properties.asciidoc]: There is one .setConverterFactory to much :)

Comment (anasmi) [reply documentation/datamodel/datamodel-properties.asciidoc]: uuups, indeed Thanks! :)

**Review by OlliTietavainenVaadin — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11223#-:-LOw6_L65Ly-OPbFzDbj:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @knoobie)

**Review by anasmi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11223)* status: all files reviewed, 1 unresolved discussion (waiting on @knoobie)

___
*[documentation/datamodel/datamodel-properties.asciidoc, line 302 at r1](https://reviewable.io/reviews/vaadin/framework/11223#-LOGvzi564AVVll6LKti-r1-302:-LOw7Irk8VCxdGG6JH6g:b-896fix) ([raw file](https://github.com/vaadin/framework/blob/d37f7052ad5cc06af47c45593424b6493a0c6e6b/documentation/datamodel/datamodel-properties.asciidoc#L302)):*
<details><summary><i>Previously, anasmi (Anastasia Smirnova) wrote…</i></summary><blockquote>

uuups, indeed. Thanks! :) 
</blockquote></details>

Done.

**Review by OlliTietavainenVaadin — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11223)* status: all files reviewed, 1 unresolved discussion (waiting on @knoobie)

**Review by knoobie — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/vaadin/framework/11223)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**